libcxx/include/__algorithm/count_if.h

```diff
@@ -10,26 +10,52 @@
 #ifndef _LIBCPP___ALGORITHM_COUNT_IF_H
 #define _LIBCPP___ALGORITHM_COUNT_IF_H
 
+#include <__algorithm/for_each.h>
 #include <__config>
 #include <__iterator/iterator_traits.h>
+#include <__iterator/segmented_iterator.h>
```

Comment:
Not attached: Please add a benchmark to show that this actually improves performance.